src/views/VerifyPrevAddress.vue

```diff
@@ -4,11 +4,12 @@
       <ProgressBar :nodeData="progressBarData"/>
       <h1>Verify you own {{alias}}@nav.community</h1>
       <div class="input-container">
-        <ListEntry><span slot="number" class="number">1</span><span slot="text" class="text small">Copy the message below</span></ListEntry>
+        <ListEntry><span slot="number" class="title number">1</span><span slot="text" class="text small">Copy the message below</span></ListEntry>
         <Copybox>signmessage {{aliasCurrentAddress}} {{alias}}@nav.community</Copybox>
```

Comment:
Copy box is breaking on small devices?

![image](https://user-images.githubusercontent.com/9244507/43806450-391b55de-9a61-11e8-9b6b-331ad5c335d3.png)

Reply:
fixed by changing font size to 13px after width < 410px